pages/unit.page.ts

```diff
@@ -7,8 +7,8 @@ class UnitPage extends Page {
         super(page);
     }
 
-    unitServicesTitle = this.page.locator('[class*="UnitCharacteristics_services"] [class*="UnitCharacteristics_title"]');
-    unitServicesTypesContainer = this.page.locator('[itemprop="services"]');
+    unitServicesTitle: Locator = this.page.locator('[class*="UnitCharacteristics_services"] [class*="UnitCharacteristics_title"]');
```

Comment:
Якщо локатори чи інші поля класу доступні публічно, варто зробити їх незмінними.

```suggestion
    readonly unitServicesTitle: Locator = this.page.locator('[class*="UnitCharacteristics_services"] [class*="UnitCharacteristics_title"]');
```